app/services/order_cycle_webhook_service.rb

```diff
@@ -11,6 +11,9 @@ def self.create_webhook_job(order_cycle, event)
     # Endpoints for coordinator owner
     webhook_endpoints = order_cycle.coordinator.owner.webhook_endpoints
 
+    # Plus unique endpoints for distributor owners (ignore duplicates)
+    webhook_endpoints |= order_cycle.distributors.map(&:owner).flat_map(&:webhook_endpoints)
```

Comment:
TIL about `|=` operator, if I understand correctly it is equivalent to :
```ruby
webhook_endpoints.union(order_cycle.distributors.map(&:owner).flat_map(&:webhook_endpoints))
```
I wonder if we should be using `union` to be more explicit on what's is happening here ?

Reply:
Yes, I think that's a good idea, it's pretty obscure. I had to look up the operator before I used it.

Reply:
Ah, except that `union` doesn't update the object itself. Also this object is array-like and doesn't support `union` directly, so it would need to be more like:

```ruby
webhook_endpoints = webhook_endpoints.to_a.union(order_cycle.distributors.map(&:owner).flat_map(&:webhook_endpoints))
```

I can think of a few other ways to write this, but I think I'll just stick with the original now...